packages/react-mutation-mapper/src/component/track/TrackPanel.tsx

```diff
@@ -25,7 +31,7 @@ type TrackPanelProps = {
 @observer
 export default class TrackPanel extends React.Component<TrackPanelProps, {}> {
     public static defaultProps: Partial<TrackPanelProps> = {
-        tracks: [TrackName.CancerHotspots, TrackName.OncoKB, TrackName.PTM],
+        tracks: [TrackName.CancerHotspots, TrackName.OncoKB, TrackName.dbPTM],
```

Comment:
Just wanted to make sure that only `dbPTM` is default and `UniprotPTM` is not?

Reply:
Correct, `dbPTM` is the default for cBioPortal.